React/CoreModules/RCTAlertController.m

```diff
@@ -32,12 +32,16 @@ - (UIWindow *)alertWindow
 
 - (void)show:(BOOL)animated completion:(void (^)(void))completion
 {
-  [self.alertWindow makeKeyAndVisible];
+  // Call self.alertWindow to ensure that it gets populated
+  UIWindow *alertWindow = self.alertWindow;
 
   // [TODO(macOS GH#774)
   // If the window is tracked by our application then it will show the alert
-  if ([[[UIApplication sharedApplication] windows] containsObject:self.alertWindow]) {
-    [self.alertWindow.rootViewController presentViewController:self animated:animated completion:completion];
+  if ([[[UIApplication sharedApplication] windows] containsObject:alertWindow]) {
+    // On iOS 14, makeKeyAndVisible should only be called if alertWindow is tracked by the application.
+    // Later versions of iOS appear to already do this check for us behind the scenes.
+    [alertWindow makeKeyAndVisible];
+    [alertWindow.rootViewController presentViewController:self animated:animated completion:completion];
```

Comment:
Sounds like we could put an iOS 14 availability check here to make it easier to remove the extra line once we drop support?

Reply:
I mean, we still want to call `[alertWindow makeKeyAndVisible]`, and it seems like it doesn't hurt for us to do this check anyway.